shapeworks_cloud/manage_workers.py

```diff
@@ -116,7 +140,7 @@ def start_all():
     client.start_instances(InstanceIds=all_ids)
 
     # Wait for startup
-    time.sleep(60)
+    time.sleep(30)
```

Comment:
Is there an impact from changing this from 60 to 30?

Reply:
The idea behind this is decreasing the window of opportunity for a race condition. The CI Action calls this function immediately prior to starting the Ansible playbook, wherein the first step is to create the lockfile to prevent the main worker from shutting down GPU workers. This window of time where we wait for startup is an opportunity for the main worker to shut down the GPU workers before the lockfile is created. However, this value is an initial guess; if this isn't enough time to wait for startup, we may need to increase this value again.